src/components/ErrorPanel/index.jsx

```diff
@@ -164,7 +168,11 @@ export default class ErrorPanel extends Component {
         )}
         disabled={!showStack}>
         <ExpansionPanelSummary
-          classes={{ disabled: classes.disabled }}
+          classes={{
+            disabled: classes.disabled,
+            expandIcon: classes.expandIcon,
+          }}
+          IconButtonProps={{ edge: false }}
```

Comment:
`IconButtonProps={{ edge: 'end' }}` then you can remove `classes.expandIcon`.

Reply:
classes.expandIcon from @withStyles, after migration arrow icon is to far from edge, I added css rules that work like in previous version
right: 8px,
position: absolute

Reply:
Ah!!! I didn't get it! Lol

Reply:
Also it's edge: "end" by default) 
I will remove IconButtonProps
The difference is 4px between v3 and v4. Is it ok?

Reply:
Yeah, I think it's ok.